mpsp/link_generator.py

```diff
@@ -0,0 +1,297 @@
+import pandas
+import csv
+import requests
+from pandas_ods_reader import read_ods
+
+def download(mes, ano):
+    if ano == '2020':
+        if mes == '01':
+            MPSP_URL_MEMB_ATIVOS = 'http://www.mpsp.mp.br/portal/page/portal/Portal_da_Transparencia/Contracheque/Membros_ativos/Tabela%20I%20memb012020.ods'
```

Comment:
É possível enxergar um padrao nos endpoints, sugiro agrupar os endpoints que possuem uma lógica para mês e ano. 
Ex:
Todos os meses de 2020, seguem um padrao, logo quando o ano for 2020, já redirecione o crawler para aquele mês padrao. 
Colocar hardCoded nao é uma boa prática e nao soluciona o problema nem temporariamente, imagine que dessa forma, todo mês alguém ia ter que adicionar um novo endpoint no arquivo.